index.bs

```diff
@@ -332,6 +332,11 @@ dimensions about the current Picture-in-Picture window.
 
 # Security considerations # {#security-considerations}
 
+<em>This section is non-normative.</em>
+
+The API only applies on {{HTMLVideoElement}} to punt security issues around
+PIP-ing arbitrary HTML content and handling events.
```

Comment:
The API applies only to {{HTMLVideoElement}} in order to start on a minimal viable product that has limited security issues. Later versions of this specification may allow PIP-ing arbitrary HTML content.